tools/postprocess.py

```diff
@@ -141,13 +141,15 @@ def on_post_build(env):
                             + f"<h1{match.group(1)}>{match.group(2)}</h1>"
                             + html_content[match.end():])
                         file_was_changed = True
+                    #"""
```

Comment:
?

Reply:
Just a big fat comment to keep this code safe.